tests/by-util/test_sort.rs

```diff
@@ -978,7 +978,7 @@ fn test_sigpipe_panic() {
     // The "Broken pipe" error should be silently ignored.
     child.close_stdout();
     assert_eq!(
-        String::from_utf8(child.wait_with_output().unwrap().stderr),
+        String::from_utf8(Vec::from(child.wait().unwrap().stderr())),
```

Comment:
Isn't this just `stdout_empty()`?

Reply:
Nice! The whole assert_eq! can be exchanged with `child.wait().unwrap().no_stderr()`